crates/mipsy_web/src/state/update.rs

```diff
@@ -88,7 +88,7 @@ pub fn handle_response_from_worker(
                 }))
             }
 
-            _ => unreachable!("cannot update binary if there is no binary"),
```

Comment:
why?

Reply:
I don't remember the circumstances now but I ran into it somehow during testing